data/hai/hai reveal 5 philosophers.txt

```diff
@@ -371,11 +461,14 @@ mission "Hai Reveal [B05-long] The Red Mystery"
-			`	She looks at the many tabs displaying the combining the information you've brought and she's found. "You know, if there's something special needed. When I was searching things up you might remember that I found an online community. I cam across a man who might, just maybe, know something useful. He claims to have found something odd related to cases which are present in the set in your data, and one of the other cases in your set is the disappearance of his wife which is where his motivation comes from.`
-			`	"His theory is... well it's a bit crackpot for the local authorities. I wouldn't have believed it either were it not for the vastly superior swathe of information we now have at our disposal, but it might be what you need. I think you should go seek him out on Tarazed."`
-			`	She rummages around and then after a moment presents you with a data stick. "This has the information he gave me to follow him up. I convinced him to trust me so hopefully he's receptive to you. It also has all the data that explicitly identifies all the ship ID's that are associated with our suspicious cases. That's the best I've got for you though. If this doesn't pan out we'll need to find different assistance."`
+			`At length you return again to tell Giti about the system with the unstable wormhole in it. You aren't required to land elsewhere on the planet this time, so it's a shorter, if equally crowded, journey to her office.`
+			`	After the both of you spend some time pouring over things for a while, she concludes that they must have some sort of special feature they're using to access it. "It just doesn't make sense that there'd be a system there, that they have coordinates to, that they're not using. Especially when we can't find them anywhere else."`
+			choice
+				`	"Do you have any other leads I can follow?"`
+				`	"Even if I've found them, I don't know what they're doing or how. Any ideas?"`
+			`	She looks at the many tabs displaying the information you've brought or she's found. "You remember that online community on unexplained kidnappings? I came across a man who might, just maybe, know something useful. He claims to have found something odd related to cases in the set in your data, and one of the cases in your set is the disappearance of his wife, which is where his motivation comes from.`
+			`	"His theory is... well it's a bit crackpot for the local authorities. I wouldn't have believed it either were it not for the vastly superior swathe of information we now have at our disposal, but it might be what you need. I think you should go seek him out on <planet>."`
```

Comment:
```suggestion
			`	"His theory is... well, it's a bit crackpot for the local authorities. I wouldn't have believed it either, were it not for the vastly superior swathe of information we now have at our disposal, but it might be what you need. I think you should go seek him out on <planet>."`
```